### ISSUE TITLE
Is this Steam only?

### ISSUE THREAD

**Issue by TheQuickFox** [sultim-t/vkquake-rt#4]
I have the original CD-ROM version of Quake. Can I use this mod with it? Or does it only work with the Steam version of Quake?

**Comment by Calinou** [comment]
Yes, you can use it as long as you find a way to extract `pak0.pak` and `pak1.pak` from the CD.

**Comment by BrendaEM** [comment]
Thanks for trying, but it didn't work on my original, full, non-shareware Quake (1).
It asks for the Steam folder, asks about the resolution, and dies.

**Comment by coxsk8** [comment]
Instructions very clear,works at once. I have all paid quake versions, steam included.

I'm very thankful for this work, cant believe it. You make an old school guy very happy, cant stop playing it, its a whole new thing. really!!

i5 16gb rtx3070
